src/components/check.jsx

```diff
@@ -0,0 +1,296 @@
+import React, { useState, useEffect, useRef } from 'react';
```

Comment:
This entire file seems redundant, but we'll deal with it later

Reply:
resolved i removed it i wasn't used